src/node_kernels/SDRSSTDESNodeKernel.C

```diff
@@ -0,0 +1,120 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+#include "node_kernels/SDRSSTDESNodeKernel.h"
```

Comment:
@rcknaus I think this file is missing in this pull request. When building, I'm getting an error.

Reply:
I forgot to include it.  I just pushed a fix to master.